newIDE/app/src/UI/TreeView/TreeViewRow.js

```diff
@@ -381,6 +387,12 @@ const TreeViewRow = <Item: ItemBaseAttributes>(props: Props<Item>) => {
 
           const rightButton = node.rightButton;
 
+          const isMenuShown =
```

Comment:
I prefer to use `shouldDisplayMenu`, wdyt?

Reply:
agree